umh.docs.umh.app/content/en/docs/features/connectivity/introduction-asset-management.md

```diff
@@ -4,51 +4,60 @@ content_type: task
 description: Introduction to IIoT Connectivity and Asset Management in UMH.
 weight: 1000
 edition: community
-draft: true
+draft: false
 ---
 
-## Introduction to IIoT Connectivity and Asset Management in UMH
 
 In IIoT, connectivity involves extracting and contextualizing data from various systems into the Unified Namespace. There's no universal solution; a blend of different tools is required, tailored to specific technical and organizational challenges.
 
-## Technical Challenges Examples:
+## When should I use it?
+Integrating diverse techniques presents challenges, involving non-standard connections, specific communication protocols, and more. Additionally, mismatches between IT and OT technology may arise. Detailed examples of these challenges are provided below. Asset Management in UMH is designed to address these issues.
+
+
+### Technical Challenges Examples:
```

Comment:
do we need it here? the technical challenged get through the numbering / bullet list a big part of the entire page, but it should be in my opinion only a side quest.